sqlite3/lib/src/common/constants.dart

```diff
@@ -507,6 +507,12 @@ class SqlFunctionFlag {
   static const SQLITE_INNOCUOUS = 0x000200000;
 }
 
+enum LoadableExtension { spellfix1 }
+
+const loadableExtensionEntrypoints = {
+  LoadableExtension.spellfix1: 'sqlite3_spellfix_init'
+};
```

Comment:
Conceptually, I feel like the `sqlite3` package shouldn't define these constants. spellfix isn't part of the generic default sqlite3 build that this package is providing Dart APIs for. Since this is an advanced feature, I think we can make `ensureExtensionLoaded` take the entrypoint symbol as a direct string and avoid the `LoadableExtension` enum.

In the `sqlite3_flutter_libs` package, which is guaranteed to have spellfix1 available, we could define helpers like this:

```dart
enum LoadableExtension {
  spellfix1('sqlite3_spellfix_init');

  final String _entrypoint;
  const LoadableExtension(this._entrypoint);
}

extension LoadKnownExtension on Sqlite3 {
  void loadKnownExtension(LoadableExtension e) => ensureExtensionLoaded(e._entrypoint);
}
```

What do you think?